lib/internal/worker/io.js

```diff
@@ -400,19 +417,40 @@ class BroadcastChannel extends EventTarget {
       throw new DOMException('Message could not be posted.');
   }
 
+  // TODO(@jasnell): The ref() method is Node.js specific and not part
+  // of the standard BroadcastChannel API definition. Typically we shouldn't
+  // extend Web Platform APIs with Node.js specific methods but ref and unref
+  // are a bit special. We may want to revisit this, however.
```

Comment:
Tbh, I don't think we'd want to revisit this. The concept doesn't make sense in browsers, but in Node.js it's a necessary part of process lifetime management.